### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

File: blockchain/addressGenerator.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type addressGenerator struct {
 	hasher         hashing.Hasher
 }
 
-// NewAddressGenerator creates an address generator instance
+// NewAddressGenerator will creates a address generator instance
```
Comment (bogdan-rosianu) [inline blockchain/addressGenerator.go:33]: will create*
an* address

File: blockchain/proxy.go [hunk 2]
```diff
@@ -165,10 +167,51 @@ func (ep *elrondProxy) SendTransaction(tx *data.Transaction) (string, error) {
...
+}
+
+func (ep *elrondProxy) postProcessSendMultipleTxsResult(response *data.SendTransactionsResponse) ([]string, error) {
+	txHashes := make([]string, 0, len(response.Data.TxHashes))
+	indexes := make([]int, 0, len(response.Data.TxHashes))
+	for index := range response.Data.TxHashes {
+		indexes = append(indexes, index)
+	}
+
+	sort.Slice(indexes, func(i, j int) bool {
```
Comment (bogdan-rosianu) [inline blockchain/proxy.go:200]: last time I checked, the order was already correct. do we need the sorting?

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: blockchain/proxy.go (hunk 2 above)
Comment (iulianpascalau) [reply blockchain/proxy.go:200]: Yes, the response comes as a map. Will be thoroughly tested in next PRs

File: blockchain/addressGenerator.go (hunk 1 above)
Comment (iulianpascalau) [reply blockchain/addressGenerator.go:33]: fixed

**Review by bogdan-rosianu — DISMISSED** [review]

File: blockchain/proxy.go [hunk 3]
```diff
@@ -165,10 +167,51 @@ func (ep *elrondProxy) SendTransaction(tx *data.Transaction) (string, error) {
...
+		return indexes[i] < indexes[j]
+	})
+
+	for _, idx := range indexes {
+		txHashes = append(txHashes, txHashes[idx])
```
Comment (SebastianMarian) [inline blockchain/proxy.go:205]: response.Data.TxHashes[idx] instead txHashes[idx]

Comment (iulianpascalau) [reply blockchain/proxy.go]: done, thanks

**Review by bogdan-rosianu — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**